readthedocs_ext/readthedocs.py

```diff
@@ -70,7 +71,18 @@ def update_body(app, pagename, templatename, context, doctree):
         # Only insert on our HTML builds
         return
 
-    if theme_css not in app.builder.css_files:
+    inject_css = True
+
+    # After v0.3.0 of the sphinx theme, the theme CSS should not be injected
+    # This decouples the theme CSS (which is versioned independently) from readthedocs.org
+    if theme_css.endswith('sphinx_rtd_theme.css'):
```

Comment:
I think it's safer to check if `context['html_theme'] == 'sphinx_rtd_theme'`. The current check does not work if `app.builder.name == 'readthedocssinglehtmllocalmedia'` (see line 60).

Reply:
Nothing is necessary with `readthedocssinglehtmllocalmedia` and this change purposely doesn't apply there. That builder doesn't use the theme CSS on media.readthedocs.org.

Reply:
Alrighty 👍